Cargo.toml

```diff
@@ -12,7 +12,7 @@ crate-type = ["cdylib"]
 [dependencies]
 log = "0.4"
 itertools = "0.8"
-ironoxide = "0.13.0"
+ironoxide = "0.14"
```

Comment:
Reviewers: Is there a reason we previously depended on "0.13.0" instead of "0.13"?

Reply:
We usually use ~0.14.0 to get >=0.14.0 and < 0.15.0. I'm not sure why it was an exact before.

Reply:
Changed to "~0.14.0"